README.md

```diff
@@ -36,7 +36,8 @@ export MAPBOX_ACCESS_TOKEN=my.token
 # Commands
 
 * Tileset Sources
-  * [`add-source`](#add-source)
+  * [`upload-source`](#add-source)
+  * *deprecated* [`add-source`](#add-source)
```

Comment:
Anchor should be deprecated-add-source I think

Reply:
👍